_config.yml

```diff
@@ -0,0 +1,3 @@
+collections:
+  jobs:
+    output: true
```

Comment:
This looks good! Can we also customize the permalink, so that the `.html` extension is not in the generated URL? See https://jekyllrb.com/docs/permalinks/#collections

Reply:
Ooh, that's nice to not have the extension in the URL. Addressed in 1d353e07d8c3e8d12a33774a7deba59ac7ee64ed